editor/src/components/canvas/canvas-strategies/strategies/reparent-utils.ts

```diff
@@ -287,11 +295,8 @@ export function placeholderCloneCommands(
   return { commands: commands, duplicatedElementNewUids: duplicatedElementNewUids }
 }
 
-function rectangleSizesEqual(
-  a: MaybeInfinityCanvasRectangle | null,
-  b: MaybeInfinityCanvasRectangle | null,
-): boolean {
-  if (a == null || b == null || isInfinityRectangle(a) || isInfinityRectangle(b)) {
+function rectangleSizesEqual(a: CanvasRectangle, b: CanvasRectangle): boolean {
+  if (isInfinityRectangle(a) || isInfinityRectangle(b)) {
```

Comment:
Is this check still applicable?

Reply:
thanks for noticing, I remove the if in https://github.com/concrete-utopia/utopia/pull/4414/commits/73c06d0aea9c8707a1c12e6c52f5d95535e62935